ogcapi-features/README.md

```diff
@@ -62,12 +62,12 @@ The core OGC API - Features endpoints are shown below, with details provided in
 | Endpoint                                        | Returns                                                 | Description                                                                                                                                |
 | ----------------------------------------------- | ------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
 | `/`                                             | [Catalog](../stac-spec/catalog-spec/README.md)          | Landing page, links to API capabilities                                                                                                    |
+| `/api`                                          | OpenAPI 3.0 JSON                                        | Returns an OpenAPI description of the service from the `service-desc` link `rel` - not required to be `/api`, but the document is required |
```

Comment:
Here it doesn't say recommended?

Reply:
Hmm, OAFeat is working on 3.1 already so we should try to be future-proof, I think.

Reply:
IIRC, we were blocked for a while on some of our tooling not supporting it yet. I'll check on that again. Filed https://github.com/radiantearth/stac-api-spec/issues/213 to track updating to 3.1.

Reply:
My intent was not to update to 3.1 here, just to make it future-proof and allow it!

Reply:
Oh, I understand now. I think we should discuss that more, since I think that makes client implementation more difficult. Definitely something we should consider for the next beta/rc. https://github.com/radiantearth/stac-api-spec/issues/214